src/coreclr/jit/regset.cpp

```diff
@@ -44,6 +44,34 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+#ifdef SWIFT_SUPPORT
+    regMaskTP rsAllCalleeSavedMask;
+    regMaskTP rsIntCalleeSavedMask;
```

Comment:
Do you think it would be cleaner to initialize these here instead of in `RegSet`'s constructor below? Also, it might be nice to add a comment explaining these aren't `const` or `constexpr` because `rsClearRegsModified` (and only this method) can modify them in Swift interop scenarios.

Reply:
> Do you think it would be cleaner to initialize these here instead of in RegSet's constructor below?

sure.

> `constexpr`

Was there a specific TP improvement we saw by marking them `constexpr`? I am inclined to just make them and it is just cleaner.

```c++
regMaskTP rsAllCalleeSavedMask = RBM_CALLEE_SAVED;
regMaskTP rsIntCalleeSavedMask = RBM_INT_CALLEE_SAVED;
```

Reply:
Will check if there is any TP impact with this change.

Reply:
> Was there a specific TP improvement we saw by marking them constexpr?

This was part of a bigger change, so I'm not sure what the TP impact is of changing this. I'm curious to see as well